app/stores/AppStore.js

```diff
@@ -11,8 +11,13 @@ export default class AppStore extends Store {
     this.actions.goToRoute.listen(this._updateRouteLocation);
   }
 
+  @computed get currentRoute() {
```

Comment:
Not sure if that's the best idea because it's just an alias to the router store without any benefit?